### PR TITLE
More old python2 stuff dropping

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: lib/vsc/filesystem/oceanstor.py [hunk 1]
```diff
@@ -281,7 +277,7 @@ class OceanStorOperationError(PosixOperationError):
     pass
 
 
-class OceanStorOperations(with_metaclass(Singleton, PosixOperations)):
+class OceanStorOperations(PosixOperations,metaclass=Singleton):
```
Comment (wpoely86) [inline lib/vsc/filesystem/oceanstor.py]: @lexming any clue on the correct syntax for this?

Comment (lexming) [reply lib/vsc/filesystem/oceanstor.py]: this change looks correct to me, we use the same syntax to use custom metaclasses in `vsc-config`

**Comment by lexming** [comment]
Unit tests are positive now:
https://dev.azure.com/VUB-ICT/Directie%20ICT/_build/results?buildId=19728&view=results

**Comment by lexming** [comment]
@wpoely86 is this ready for merge or do you want to add anything else?

**Comment by wpoely86** [comment]
ready for me. You need to approve :wink:

**Review by lexming — APPROVED** [review]

**Pull request merged**